GoogleMapsApi/Engine/MapsAPIGenericEngine.cs

```diff
@@ -51,7 +50,7 @@ public static int HttpsConnectionLimit
 		private const string AuthenticationFailedMessage =
 			"The request to Google API failed with HTTP error '(403) Forbidden', which usually indicates that the provided client ID or signing key is invalid or expired.";
 
-		static MapsAPIGenericEngine()
+		static MapsApiGenericEngine()
```

Comment:
This is a breaking change! Anyway, it doesn't relate to the .NET Core upgrade.

Reply:
Would not touch it as well.